schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/common/DefaultOrganizationService.java

```diff
@@ -83,22 +83,34 @@ public Organization getOrganization(Key id) throws EntityNotFoundException, Ille
         }
     }
 
-    @Override
-    public boolean isOrganizationMember(User user) {
-        for (Organization organization : getOrganizations()) {
-            if (organization.getUsers().contains(user))
-                return true;
+    public OrganizationMember getOrganizationMember(Key id) throws EntityNotFoundException, IllegalArgumentException {
+        try {
+            return Datastore.get(organizationMemberMeta, id);
+        } catch (EntityNotFoundRuntimeException e) {
+            throw new EntityNotFoundException("OrganizationMember id=" + id);
         }
-        return false;
+    }
+
+    @Override
+    public boolean isOrganizationMember(Key userId) {
+        List<Key> result = Datastore.query(organizationMemberMeta)
+                .filter(organizationMemberMeta.userRef.equal(userId))
+                .asKeyList();
+        return !result.isEmpty();
     }
 
     @Nonnull
     @Override
-    public List<Organization> getOrganizationsForMember(User user) throws EntityNotFoundException {
```

Comment:
Sorry to be pedantic.  But can you please rename to `getOrganizationsForUser` since for group, you will need a different method.